showkase-processor/src/main/java/com/airbnb/android/showkase/processor/models/ShowkaseMetadata.kt

```diff
@@ -298,6 +292,59 @@ internal fun getShowkaseMetadataFromPreview(
     }
 }
 
+@Suppress("LongParameterList", "LongMethod")
+internal fun getShowkaseMetadataFromCustomAnnotation(
+    element: XMethodElement,
+    showkaseValidator: ShowkaseValidator,
+    annotationName: String,
+    roundEnv: XRoundEnv,
+): List<ShowkaseMetadata.Component?> {
+    val customAnnotation = element.requireAnnotationBySimpleName(annotationName)
+
+    val previewAnnotations = customAnnotation.map {
+//        it.typeElement.getAnnotations(PREVIEW_CLASS_NAME)
+        it.typeElement.requireAnnotationBySimpleName(PREVIEW_SIMPLE_NAME)
+    }.flatten()
```

Comment:
Here I ran into an issue when it comes to annotations that are in different modules. If they are in the same module, the typeElement has preview annotations. However, when they are in a different module, it never has any annotations. That list is always empty 🤔  Do you have any tips on how I can make that work @vinaygaba ? :) Is there some magic I can use here? 😅

Reply:
Maybe I can ask in the Kotlin KSP channel also if you don't know :)

Reply:
If the user is using a custom annotation like `@MyCustomPreview`, they aren't using @Preview annotations in addition to that no? We have to implicitly get it based on the previews that are originally declared in  `@MyCustomPreview`

Reply:
Yeah, I was thinking, does that mean that I will need to ask the user to declare more than just the qualified name for the custom annotation in their gradle file? Like whatever information I would get from the `@Preview` annotation?

Because what I was trying to do here was to get the preview annotations that the `@MyCustomPreview` is annotated with. I am first getting the annotation from the `XRoundEnvironment` and then I thought since I knew the qualified name of the preview annotation I would be able to get that also from the `requireAnotationBySimpleName` or from the `XRoundEnvironment` somehow. 

IMO, this was somewhat confusing. Would be very nice with a call to discuss it :)